mmseg/models/backbones/vit.py

```diff
@@ -371,7 +370,7 @@ def resize_pos_embed(pos_embed, input_shpae, pos_shape, patch_size, mode):
             1, pos_h, pos_w, pos_embed.shape[2]).permute(0, 3, 1, 2)
         pos_embed_weight = F.interpolate(
             pos_embed_weight,
-            size=[input_h // patch_size, input_w // patch_size],
+            size=[input_h, input_w],
```

Comment:
```suggestion
            size=input_shpae,
```

and `input_h, input_w = input_shpae` is redundant.